src/features/Planner/data/utils.ts

```diff
@@ -1052,9 +1052,11 @@ export function mapPlanBuckets(
 }
 
 export function saveBucket(state: State, bucket: PlanBucket) {
-    ClientId.is(bucket.id)
-        ? PlanApi.createBucket(state.activeListId!, bucket)
-        : PlanApi.updateBucket(state.activeListId!, bucket.id, bucket);
+    if (ClientId.is(bucket.id)) {
+        PlanApi.createBucket(state.activeListId!, bucket);
+    } else {
+        PlanApi.updateBucket(state.activeListId!, bucket.id, bucket);
+    }
```

Comment:
This is probably the most interesting lint change: use of ternary _without using the result_ is now an error. Have to use `if` if selecting between procedures to invoke, rather than evaluating a result.